frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-pool.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+import {
+  confirmAction,
+  createStorageClassButton,
+  emptyStateBody,
+  storagePoolDropdown,
+  storagePool,
+} from '../views/multiple-pool';
+
+describe('Test Ceph pool creation', () => {
+  // Pool var
+  const poolName: string = 'example.pool';
+  const replicaCount: string = '2';
+  const RBDProvisioner: string = 'openshift-storage.rbd.csi.ceph.com';
+
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    // cy.install();
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+    cy.clickNavLink(['Storage', 'Storage Classes']);
+    cy.byTestID(createStorageClassButton).click();
+  });
+
+  after(() => {
+    storagePool.delete(poolName);
+    cy.logout();
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  it('Check for a new pool creation', () => {
+    cy.log('Creating a new pool when cluster is in ready state');
```

Comment:
```suggestion
    cy.log('Test creation of a new pool');
```
`cy.install()` guarantees we are in a ready state.